conda_devenv/devenv.py

```diff
@@ -454,6 +454,8 @@ def write_activate_deactivate_scripts(
 
     if sys.platform == "win32":
         files = [("devenv-vars.bat", "cmd")]
+        if args.generate_ps1:
+            files += [("devenv-vars.ps1", "ps1")]
     else:
         # Linux and Mac should create a .sh
         files = [("devenv-vars.sh", "bash"), ("devenv-vars.fish", "fish")]
```

Comment:
Was it a problem to always generate them?
In linux we always generate 2 sets of scripts, one for bash and one for fish.

Reply:
I updated the PR to always generate ps1 scripts for win32